.gitignore

```diff
@@ -0,0 +1,34 @@
+.DS_STORE
+
+/dist/
+/bazel-out
+/integration/bazel/bazel-*
+*.log
+node_modules
+
+month.txt
+
+*secret.js
+
+npm-debug.log
+yarn-error.log
+
+.build-analytics
+
+/modules/rollup-test/dist/
+
+.bazelrc.user
+
+.ng-dev.user*
+
+.notes.md
+baseline.json
+
+.history
+
+.husky/
+
+reserved_code.txt
+
+.angular/cache
```

Comment:
It doesn't look like the .angular cache files got removed from the PR. Maybe try `.angular` or `.angular/`?